windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -56,7 +60,7 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 
 * The Prometheus windows_exporter used by the WMCO currently collects metrics through HTTP, so it is considered unsafe. You must ensure that only trusted users can retrieve metrics from the endpoint. The windows_exporter feature recently added support for HTTPS configuration, but this configuration has not been implemented for WMCO. Support for HTTPS configuration in the WMCO will be added in a future release.
 
-* If you have a cluster with two Windows nodes, and you create a web server deployment with two replicas, the pods each land on a Windows compute node. In this scenario, if you create a `Service` object with type `LoadBalancer`, communication with the load balancer endpoint is not reliable. To mitigate this issue, you must use Windows Server 2019 with a version 10.0.17763.1457 or earlier. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[*BZ#1905950*])
+* If you have a cluster with two Windows nodes, and you create a web server deployment with two replicas, the pods each land on a Windows compute node. In this scenario, if you create a `Service` object with type `LoadBalancer`, communication with the load balancer endpoint is not reliable. To mitigate this issue for clusters installed on vSphere, you must use Windows Server 1909 with link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. To mitigate this issue for clusters installed on AWS or Azure, you must use Windows Server 2019 with a version 10.0.17763.1457 or earlier. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[*BZ#1905950*])
```

Comment:
Based on my e2e testing [PR]([test] Set replica count based on affinity), we don't see this issue on vSphere. I suggest leaving out the vSphere line for now.